marketplace/deployer_util/config_helper.py

```diff
@@ -531,6 +539,86 @@ def nodes(self):
     return self._nodes
 
 
+class SchemaAssistedClusterCreation:
+  """Accesses top level AssistedClusterCreation constraints."""
+
+  _ASSISTED_CC_TYPE_DISABLED = "DISABLED"
+  _ASSISTED_CC_TYPE_STRICT = "STRICT"
+  _ASSISTED_CC_TYPES = [_ASSISTED_CC_TYPE_DISABLED, _ASSISTED_CC_TYPE_STRICT]
+
+  def __init__(self, dictionary):
+    self._type = None
+    self._creation_guidance = None
+    self._gke = None
+
+    self._type = dictionary.get('type', None)
+    _must_contain(self._type, self._ASSISTED_CC_TYPES,
+                  "Invalid type of AssistedClusterCreation")
+    self._creation_guidance = dictionary.get('creationGuidance', "")
```

Comment:
Use a default value of `None` to be consistent with our string default-value handling within this file.

Reply:
Done.